rules/jest.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     'jest/require-top-level-describe': 'off',
     'jest/prefer-called-with': 'off',
     'jest/prefer-expect-assertions': 'error',
+    'jest/prefer-importing-jest-globals': 'off',
```

Comment:
having this rule enabled would mean too many changes to our codebases, so disabling it